ir/src/main/scala/org/scalajs/core/ir/IRVersionNotSupportedException.scala

```diff
@@ -0,0 +1,13 @@
+package org.scalajs.core.ir
+
+import java.io.IOException
+
+class IRVersionNotSupportedException(val version: String,
+    val supported: Set[String], message: String) extends IOException(message) {
```

Comment:
I removed the case to avoid exposing the field `message` as this one should be accessed with the standard `getMessage` method.